presto-main/src/main/java/com/facebook/presto/operator/ExchangeClient.java

```diff
@@ -535,6 +542,17 @@ private static void closeQuietly(PageBufferClient client)
         }
     }
 
+    private static Optional<URI> getAsyncPageTransportLocation(URI location, boolean asyncPageTransportEnabled)
```

Comment:
Add a comment that this will eventually be removed?

Reply:
Comment added to `ExchangeClientConfig`